Cargo.toml

```diff
@@ -41,6 +42,7 @@ unsigned-varint = "0.5.0"
 
 blake2b_simd = { version = "0.5.10", default-features = false, optional = true }
 blake2s_simd = { version = "0.5.10", default-features = false, optional = true }
+blake3_ = { version = "0.3.6", default-features = false, optional = true, package = "blake3" }
 digest = { version = "0.9.0", default-features = false, optional = true }
```

Comment:
Remove the package rename

Reply:
Then should the module be renamed to something else then? From what I understand you cant have a crate and module both have the same name.

Reply:
What makes you think that?

Reply:
Some questions that were answered on StackOverflow. You are right, that's not the case, the crate can be accessed with ::blake3